examples/generative_models/jtnn/README.md

```diff
@@ -31,10 +56,25 @@ encoded nodes(atoms) and edges(bonds), and other information for model to use.
 ## Usage
 
 ### Training
-
-To start training, use `python train.py`. By default, the script will use ZINC dataset
+####ZINC
```

Comment:
I think there needs to be an empty space after #### for rendering considerations.